dataprofiler/tests/labelers/test_data_labelers.py

```diff
@@ -36,47 +36,79 @@ def test_train_method_exists(self):
         self.assertTrue(hasattr(dp, 'train_structured_labeler'))
 
         with mock.patch('dataprofiler.train_structured_labeler') as mock_obj:
-            dp.train_structured_labeler(None, None)
-            self.assertEqual(mock_obj.call_args, mock.call(None, None))
+            dp.train_structured_labeler(None, None, None)
+            self.assertEqual(mock_obj.call_args, mock.call(None, None, None))
 
     def test_accepted_inputs(self):
         with self.assertRaisesRegex(TypeError,
                                     "Input data must be either a "
                                     "`pd.DataFrame` or a `data_profiler.Data` "
                                     "and not of type `TextData`."):
-            dp.train_structured_labeler(None, None)
+            dp.train_structured_labeler(None, None, None)
```

Comment:
tbh probably only need `dp.train_structured_labeler(None)`

Reply:
just wanted to show we have three parameters there. But will change back if needed.

Reply:
changed